sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/ConversationAnalysisClient.cs

```diff
@@ -135,20 +137,156 @@ public virtual Response<AnalyzeConversationTaskResult> AnalyzeConversation(strin
+            scope.Start();
+
+            var content = new Utf8JsonRequestContent();
+            content.JsonWriter.WriteObjectValue(analyzeConversationJobsInput);
+
+            try
+            {
+                Operation<AnalyzeConversationJobState> response = await SubmitJobAsync(content, new RequestContext() { CancellationToken = cancellationToken }).ConfigureAwait(false);
+                return response;
             }
             catch (Exception ex)
             {
                 scope.Failed(ex);
                 throw;
             }
         }
+
+        /// <summary>Analyzes a conversational utterance.</summary>
+        /// <param name="input">The <see cref="GeneratedConversation"/> used for tasks input.</param>
+        /// <param name="tasks"> <see cref="AnalyzeConversationLROTask"/> defines the tasks to be used.</param>
+        /// <param name="cancellationToken">An optional <see cref="CancellationToken"/> to cancel the request.</param>
+        /// <exception cref="ArgumentNullException"><paramref name="input"/> or <paramref name="tasks"/> or is null.</exception>
+        /// <exception cref="RequestFailedException">The service returned an error. The exception contains details of the service error.</exception>
+        public virtual Operation<AnalyzeConversationJobState> StartAnalyzeConversation(IEnumerable<GeneratedConversation> input, IEnumerable<AnalyzeConversationLROTask> tasks, CancellationToken cancellationToken = default)
+        {
+            Argument.AssertNotNullOrEmpty(input, nameof(input));
+            Argument.AssertNotNull(tasks, nameof(tasks));
+
+            MultiLanguageConversationAnalysisInput multiLanguageConversationAnalysisInput = new MultiLanguageConversationAnalysisInput(input);
+
+            AnalyzeConversationJobsInput analyzeConversationJobsInput = new AnalyzeConversationJobsInput(multiLanguageConversationAnalysisInput, tasks);
+
+            using DiagnosticScope scope = ClientDiagnostics.CreateScope($"{nameof(ConversationAnalysisClient)}.{nameof(StartAnalyzeConversation)}");
+            scope.Start();
+
+            var content = new Utf8JsonRequestContent();
+            content.JsonWriter.WriteObjectValue(analyzeConversationJobsInput);
+
+            try
+            {
+                Operation<AnalyzeConversationJobState> response = SubmitJob(content, new RequestContext() { CancellationToken = cancellationToken });
+                return response;
+            }
+            catch (Exception ex)
+            {
+                scope.Failed(ex);
+                throw;
+            }
+        }
+
+        private static AnalyzeConversationJobState ConvertResponseToResult(Response response)
+        {
+            using var document = JsonDocument.Parse(response.ContentStream);
+            return AnalyzeConversationJobState.DeserializeAnalyzeConversationJobState(document.RootElement);
+        }
```

Comment:
Nit (we could do it later): I'd rather put all these conversion methods into partial classes that need them, like documented in https://github.com/Azure/azure-sdk-for-net/blob/main/doc/DataPlaneCodeGeneration/DeveloperDrivenEvolution.md#create-the-models.